api/src/main/java/com/valtech/aem/saas/api/indexing/PathExternalizer.java

```diff
@@ -0,0 +1,34 @@
+package com.valtech.aem.saas.api.indexing;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Service interface for externalizing resource path.
+ */
+public interface PathExternalizer {
```

Comment:
do we have a default implementation in place? 
I was expecting the Day CQ Externalizer to be used by default

Reply:
The cq externalizer is referenced/used in PageIndexUpdateHandler